pyproject.toml

```diff
@@ -23,6 +23,7 @@ pylint = "^2.7.2"
 autopep8 = "^1.5.5"
 yamllint = "^1.26.0"
 mypy = "^0.812"
+rope = "^0.18.0"
```

Comment:
I hate that VSCode does this... but IDK if there's a way to fix it?

Reply:
delete it?

Reply:
Mine doesn't do this... you delete it